### PR TITLE
intro: change byline to "by al."

### PR DESCRIPTION
**Pull request by justinmk**
Several people have suggested that the "by Bram" byline is misleading, it implies that Bram is actively involved with the project. Up to now we left it as an homage.

Maybe @brammool can say what he prefers.

### CODE REVIEW

**Comment by brammool** [comment]
Justin M. Keyes wrote:

> Several people have suggested that the "by Bram" byline is misleading,
> it implies that Bram is actively involved with the project. Up to now
> we left it as an homage.
> 
> Maybe @brammool can say what he prefers.

Putting my name first would indeed be misleading.  You can add something
like "based on Vim by Bram Moolenaar", but perhaps not on the intro
screen.

I can't remember seeing "by al." somewhere, looks strange to me.
Doesn't it mean "by others"?


-- 
hundred-and-one symptoms of being an internet addict:
129. You cancel your newspaper subscription.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///

**Comment by justinmk** [comment]
Thanks for the comment @brammool .

> You can add something like "based on Vim by Bram Moolenaar", but perhaps not on the intro screen.

I've added a note to `:help nvim`. Existing notes in intro.txt and help.txt will remain.

> I can't remember seeing "by al." somewhere, looks strange to me. Doesn't it mean "by others"?

Yes.

File: src/nvim/version.c [hunk 1]
```diff
@@ -2709,7 +2709,7 @@ void intro_message(int colon)
   static char *(lines[]) = {
     N_(NVIM_VERSION_LONG),
     "",
-    N_("by Bram Moolenaar et al."),
+    N_("by al."),
```
Comment (jamessan) [inline src/nvim/version.c]: `by al.` is indeed strange.  I don't think that's going to make sense outside of the context of it previously being `et al.`.  What about either

* > by Neovim contributors
* > based on Vim

Comment (justinmk) [inline src/nvim/version.c]: It's a reference to the vim message. I like that it's strange. Only me?

Comment (fwalch) [inline src/nvim/version.c]: My first impulse was to suggest to remove the message completely, as saying "made by someone" resp. "Neovim, made by the Neovim people" seems quite redundant.

In general, I like the idea of keeping it as a reference, but I only "got it" after reading your comment (and seeing the diff of old/new message on Github), so I'm not sure.

**Comment by justinmk** [comment]
Can always change it later if pitchfork limit is reached.

**Pull request merged**